fpm.toml

```diff
@@ -11,6 +11,10 @@ toml-f.rev = "54686e45993f3a9a1d05d5c7419f39e7d5a4eb3f"
 M_CLI2.git = "https://github.com/urbanjost/M_CLI2.git"
 M_CLI2.rev = "7264878cdb1baff7323cc48596d829ccfe7751b8"
 
+[preprocess]
+[preprocess.cpp]
+macros=["FPM_VERSION={version}"]
+
```

Comment:
The following update passes version from the manifest into fpm as a preprocessed macro @awvwgk. I like this way, but it cannot be built (fpm 0.3.0 used for bootstrapping). Should we update fpm to 0.7.0 in the bootstrapping action?